src/webrtc/call.ts

```diff
@@ -1640,9 +1640,14 @@ export class MatrixCall extends EventEmitter {
     }
 
     queueCandidate(content: RTCIceCandidate) {
-        // Sends candidates with are sent in a special way because we try to amalgamate
-        // them into one message
-        this.candidateSendQueue.push(content);
+        // We partially de-trickle candidates by waiting for `delay` before sending them
+        // amalgamated, in order to avoid sending too many m.call.candidates events and hitting
+        // rate limits in Matrix.
+        // In practice, it'd be better to remove rate limits for m.call.*
+        // ensure the 'ICE complete' dummy event never leaks out into m.call.candidates
```

Comment:
We deliberately send this to signal the end of candidates to the other side (often it's important, eg. if we're bridging to something without trickle ICE and the bridge has to wait for all the candidates).